helpers/auth/users.py

```diff
@@ -0,0 +1,137 @@
+test_user = {
+    'values': [
```

Comment:
Since most information contained in this file about the user should be confidential, do you think it is not safe to expose it having in mind that this repository is public?

Reply:
@Bonifase This has been resolved.